src/main/java/org/testng/IInvokedMethodListener.java

```diff
@@ -2,7 +2,8 @@
 
 /**
  * A listener that gets invoked before and after a method is invoked by TestNG. This listener will
- * only be invoked for configuration and test methods.
+ * be invoked for configuration and test methods irrespective of whether they passe/fail or
+ * get skipped.
```

Comment:
Not sure for skipped. I will let @cbeust takes the decision.

Reply:
@juherr - The documentation is reflecting the current behavior

Reply:
I think this PR should be merged with #1884

Reply:
I will come back and alter the javadocs accordingly once I take up https://github.com/cbeust/testng/issues/1892